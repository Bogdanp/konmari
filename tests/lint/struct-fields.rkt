#lang racket/base

(provide
 user
 user-username)

(struct user (username password))
