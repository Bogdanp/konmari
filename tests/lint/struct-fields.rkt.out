/struct-fields.rkt:7:24:warning:identifier 'user-password' is never used
